### PR TITLE
Changed count query for PG stats

### PR DESCRIPTION
**Pull request by ethervoid**
This should be considered a fix. I've put a count(*) that is a very expensive. With this PR it's changed to use the PG statistics.

**This PR fixes issue #4478**

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5024/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5025/))

**Comment by ethervoid** [comment]
Please @juanignaciosl CR this PR (its related to other you reviewed). Thank you.

File: services/importer/lib/importer/job.rb [hunk 1]
```diff
@@ -47,6 +47,16 @@ def concealed_pg_options
...
       end
 
+      def rows_number
+        rows = db.fetch(%Q{SELECT n_live_tup as num_rows
+                     FROM pg_stat_user_tables
+                     WHERE schemaname = '#{@schema}'
+                     AND relname = '#{table_name}';}).first
+        return (!rows.nil? && rows.has_key?(:num_rows)) ? rows[:num_rows] : nil
+      rescue
+        return 0
```
Comment (juanignaciosl) [inline services/importer/lib/importer/job.rb]: If there's a known reason for this to fail "often" you should add a comment. If it's not, you should send a debug trace (`CartoDB.notify_debug(...`) so we can detect what happens and improve.

File: services/importer/lib/importer/job.rb [hunk 2]
```diff
@@ -47,6 +47,16 @@ def concealed_pg_options
         pg_options.reject { |key, value| key.to_s == 'password' }
       end
 
+      def rows_number
+        rows = db.fetch(%Q{SELECT n_live_tup as num_rows
+                     FROM pg_stat_user_tables
+                     WHERE schemaname = '#{@schema}'
+                     AND relname = '#{table_name}';}).first
```
Comment (juanignaciosl) [inline services/importer/lib/importer/job.rb]: Not really important, but we don't use to add `;` at the end of non-DDL SQL queries. If you're making any changes to this PR please delete it. If you're not, forget this comment :)

File: services/importer/lib/importer/job.rb [hunk 3]
```diff
@@ -47,6 +47,16 @@ def concealed_pg_options
...
+        return (!rows.nil? && rows.has_key?(:num_rows)) ? rows[:num_rows] : nil
```
Comment (juanignaciosl) [inline services/importer/lib/importer/job.rb]: Another comment you can safely skip: you can write this in a more concise way with `fetch`: 
`rows.nil? ? nil : rows.fetch(:num_rows, nil)`

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5029/))

**Comment by ethervoid** [comment]
All changes done. Thank you @juanignaciosl

**Pull request merged**